xhtml2pdf/parser.py

```diff
@@ -569,6 +569,9 @@ def pisaLoop(node, context, path=None, **kw):
             value = str(context.cssAttr["-pdf-keep-in-frame-mode"]).strip().lower()
             if value in ("shrink", "error", "overflow", "truncate"):
                 keepInFrameMode = value
+            else:
+                keepInFrameMode = "shrink"
+            # Added because we need a default value.
```

Comment:
Sorry, just deleted my comment because this comment is below what it describes... it's better to comment above the code :)